internal/node/node.go

```diff
@@ -0,0 +1,75 @@
+package node
+
+import (
+	"context"
+	"fmt"
+	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+type Node interface {
+	IsNodeSchedulable(node *v1.Node) bool
```

Comment:
Since this method isn't used anywhere else besides withing the package, and since it doesn't use the object it is implemented for, then it shouldn't be an interface method.

Instead, it should be a non-exported function of the `node` package `isNodeSchedulable`

Reply:
@ybettan we also use it in internal/filter.go line 53

Reply:
Oh. I am sorry. I missed that one. Then we should keep it as an interface method as you already did :)